examples/rxdirs.cf

```diff
@@ -0,0 +1,59 @@
+###############################################################################
+#+begin_src cfengine3
+bundle agent __main__
+# @brief Illustrating the behavior of rxdirs in perms bodies. Note, by default, when read is requested for a directory, execute is given as well.
+{
```

Comment:
Also note that by default if `rxdirs` attribute is not specified then it defaults to true. This is clear after reading the entire policy or maybe looking at the docs, but not as this lone example. Does this example replace the one here? 

https://docs.cfengine.com/docs/3.18/reference-promise-types-files.html#rxdirs

Reply:
Yes, I intend to replace that useless snippet with this example.

Reply:
This is in the docs already so example is good to go. :+1: